### PR TITLE
Introduce `minimize()` public interface

### PR DESCRIPTION
**Pull request by EgorovEI**
This method minimizes Glia screens to the Bubble. It might be used by integrators during handling deep links. 
Added tests for `minimize()` and `resume()` methods.

MOB-2607

### CODE REVIEW

File: TestingApp/ViewController/ViewController.swift [hunk 1]
```diff
@@ -155,6 +155,7 @@ class ViewController: UIViewController {
 
 extension ViewController {
     func presentSettings() {
+        Glia.sharedInstance.minimize()
```
Comment (rasmustautsglia) [inline TestingApp/ViewController/ViewController.swift:158]: What is the purpose of this here?

**Review by yurii-glia — APPROVED** [review]

File: TestingApp/ViewController/ViewController.swift (hunk 1 above)
Comment (EgorovEI) [reply TestingApp/ViewController/ViewController.swift]: Thank you, good catch. It's needed to minimize engagement view before handling `glia://widgets/settings` deep link.
I moved to more suitable place, SettingsDeeplingHandler. 95891e12de8d30c53a07767fbcca81b62bf4e64d

**Review by rasmustautsglia — APPROVED** [review]

**Comment by EgorovEI** [comment]
!squash

File: GliaWidgets/Public/Glia/Glia.swift [hunk 2]
```diff
@@ -138,6 +138,13 @@ public class Glia {
         startObservingInteractorEvents()
     }
 
+    /// Minimizes engagement view if ongoing engagement exists.
+    /// Use this function for hiding the engagement view programmatically during ongoing engagement.
+    /// If do so, the operator bubble appears.
```
Comment (gersonnoboa) [inline GliaWidgets/Public/Glia/Glia.swift:143]: `If you do so` maybe? I don't know exactly what this line is trying to say.

**Review by gersonnoboa — APPROVED** [review]

**Review by igorkravchenko — APPROVED** [review]

File: GliaWidgets/Public/Glia/Glia.swift (hunk 2 above)
Comment (EgorovEI) [reply GliaWidgets/Public/Glia/Glia.swift]: 1d9a58150126c03cd6034fff3bc17a68c370edcb

**Comment by EgorovEI** [comment]
!squash

**Pull request merged**